Changelog.md

```diff
@@ -56,6 +61,34 @@
 - [#393]: Now `BytesStart::name()` and `BytesEnd::name()` returns `QName`, and
   `BytesStart::local_name()` and `BytesEnd::local_name()` returns `LocalName`
 
+- [#191]: Remove unused `reader.decoder().decode_owned()`. If you ever used it,
+  use `String::from_utf8` instead (which that function did)
+- [#191]: Remove `*_without_bom` methods from the `Attributes` struct because they are useless.
+  Use the same-named methods without that suffix instead. Attribute values cannot contain BOM
+- [#191]: Remove `Reader::decode()` and `Reader::decode_without_bom()`, they are replaced by
+  `Decoder::decode()` and `Decoder::decode_with_bom_removal()`.
+  Use `reader.decoder().decode_*(...)` instead of `reader.decode_*(...)` for now.
+  `Reader::encoding()` is replaced by `Decoder::encoding()` as well
+- [#191]: Remove poorly designed `BytesText::unescape_and_decode_without_bom()` and
+  `BytesText::unescape_and_decode_without_bom_with_custom_entities()`. Although these methods worked
+  as expected, this was only due to good luck. They was replaced by the
+  `BytesStartText::decode_with_bom_removal()`:
+  - conceptually, you should decode BOM only for the first `Text` event from the
+    reader (since now `StartText` event is emitted instead for this)
+  - text before the first tag is not an XML content at all, so it is meaningless
+    to try to unescape something in it
+
+- [#180]: Eliminate differences in the decoding API when feature `encoding` enabled and when it is
+  disabled. Since then signatures of functions are the same regardless of the feature enabled and
+  no more replacements does for invalid characters when feature is enabled. An error will be returned
+  instead, as for the case when `encoding` is disabled.
+  Namely, if `encoding` feature is disabled, `Result<Cow<&str>>` is returned instead of `Result<&str>`.
```

Comment:
Previously, if the `encoding` feature was enabled, decoding functions would return `Result<Cow<&str>>` while without this feature they would return `Result<&str>`.  With this change, only `Result<Cow<&str>>` is returned regardless of the status of the feature.